modules/virt-about-workload-updates.adoc

```diff
@@ -2,26 +2,53 @@
 //
 // * virt/upgrading-virt.adoc
 
+:_content-type: CONCEPT
 [id="virt-about-workload-updates_{context}"]
 = About workload updates
 
-* When you upgrade {VirtProductName}, virtual machine workloads including `libvirt`, `virt-launcher`, and `qemu` update automatically if they support live migration.
-+
+When you upgrade {VirtProductName}, virtual machine workloads including `libvirt`, `virt-launcher`, and `qemu` update automatically if they support live migration.
+
 [NOTE]
 ====
 Each virtual machine has a `virt-launcher` pod that runs the virtual machine
-instance. The `virt-launcher` pod runs an instance of `libvirt`, which is
-used to manage the virtual machine process.
+instance (VMI). The `virt-launcher` pod runs an instance of `libvirt`, which is
+used to manage the virtual machine (VM) process.
 ====
 
-** There are two available workload update methods: `LiveMigrate` and `Evict`. By default, only `LiveMigrate` is enabled, which means that:
+You can configure how workloads are updated by editing the `spec.workloadUpdateStrategy` stanza of the `HyperConverged` custom resource (CR). There are two available workload update methods: `LiveMigrate` and `Evict`.
+
+* Because the `Evict` method shuts down VMI pods, only the `LiveMigrate` update strategy is enabled by default.
+
+When `LiveMigrate` is the only update strategy enabled:
+
+* VMIs that support live migration are migrated during the upgrade process. The VM guest moves into a new pod with the updated components enabled.
+
+* VMIs that do not support live migration are not disrupted or updated.
+
+** If a VMI has the `LiveMigrate` eviction strategy but does not support live migration, it is not updated.
+
+If you enable both `LiveMigrate` and `Evict`:
+
+* VMIs that support live migration use the `LiveMigrate` update strategy.
 
-*** Virtual machine instances (VMIs) that support live migration are migrated during the upgrade process.
+* VMIs that do not support live migration use the `Evict` update strategy. If a VMI is controlled by a `VirtualMachine` object that has a `runStrategy` value of `always`, a new VMI is created in a new pod with updated components.
```

Comment:
@davidvossel please keep me honest on this: with `LiveMigrate` and `Evict`, a VM configured with live migration is never going to be evicted even if the VMI continuously fails to be migrated for any reason. Right?

Reply:
under normal operation, a VMI set with `EvictionStrategy: LiveMigrate` will not shutdown during eviction for any reason. The VMI will either be live migrated, or in the event that live migration isn't possible the VMI will block the eviction indefinitely.